eng/common/scripts/job-matrix/README.md

```diff
@@ -116,45 +188,30 @@ Example:
 }
 ```
 
-#### include
-
-The `include` field defines any number of matrices to be appended to the base matrix after processing exclusions.
+### include
 
-```
-# matrix entry format:
-{
-    "a": 1,
-    "b": 2,
-    "c": 3,
-}
+The `include` field defines any number of matrix combinations to be appended to the base matrix after processing exclusions.
 
-# Include field in a matrix config
-{
-    "include": [
-        {
-            "a": 1,
-            "b": 2
-        }
-    ]
-}
-```
+The value of `include` key is an array of objects, where each object represents one combination to add to all the combinations generated from the matrix.
```

Comment:
I think it's important to call out here that each include entry itself is a matrix, e.g. you can do:

```
    "include": [
        {
            "a": [1,2],
            "b": [1,2,3,4,5]
        }
    ]
```

Reply:
Oof, big miss on my side! ;) Addressed. Please have a look:

https://github.com/Azure/azure-sdk-tools/pull/5023/commits/9256553557cab2a7a7547c7f24a8c80a5981dd0c

Thanks!